src/load/azext_load/data_plane/utils/utils.py

```diff
@@ -347,6 +358,24 @@ def convert_yaml_to_test(data):
                 new_body["passFailCriteria"]["passFailMetrics"][metric_id][
                     "requestName"
                 ] = name
+    if data.get("autoStop") is not None:
+        if (isinstance(data["autoStop"], str)):
+            new_body["autoStopCriteria"] = {
+                "autoStopDisabled": True,
+            }
+        else:
+            error_rate = data["autoStop"].get("errorPercentage", Decimal(90.0))
+            time_window = data["autoStop"].get("timeWindow", 60)
+            new_body["autoStopCriteria"] = {
+                "autoStopDisabled": False,
+                "errorRate": error_rate,
+                "errorRateTimeWindowInSeconds": time_window,
+            }
+    if (new_body.get("autoStopCriteria", {}).get("autoStopDisabled") is True):
+        logger.warning(
+            "Auto stop is disabled. Error rate and time window will be ignored. "
+            "This can lead to incoming charges for an incorrectly configured test."
```

Comment:
Please get the string reviewed by Saloni or match it with the portal experience.

Reply:
I took this from the portal experience, but looks like there's been a recent change on the portal itself, only the 2nd part of the warning is now being shown in portal, @saloniagrawal1997, can you please confirm on the text here
![image](https://github.com/user-attachments/assets/d9140d05-5f5c-43aa-ade1-1b38bec51613)

Reply:
I think the "Auto stop is disabled. Error rate and time window will be ignored. "
            "This can lead to incoming charges for an incorrectly configured test." is good because they might specify error rate and time window in CLI using auto-stop command. Lets proceed with this. Not sure about the extra space after "ignored. "

Reply:
the space is needed because without it the text would appear like this: ...will be ignored.This can lead to.....

Reply:
proceeding as stated